src/parseTools.mjs

```diff
@@ -731,14 +731,22 @@ function makeEval(code) {
   return ret;
 }
 
-export const ATMAINS = [];
-
 export const ATINITS = [];
 
+// Add code to run after the Wasm module is loaded, but before static
+// constructors and main (if applicable). The code will be executed before the
+// runtime `__ATINIT__` callbacks.
 function addAtInit(code) {
   ATINITS.push(code);
 }
 
+export const ATPOSTINITS = [];
+
+// Add code to run after static constructors, but before main (if applicable).
```

Comment:
This change looks great!   The only think i'm not sure about is the naming of this new list of actions.

Reading this comment made me think that since the primary defining factor seems to be related to static ctors, perhaps we should call it something with that in the name?  `ATCTORS` ?  `ATPOSTCTORS`?  I'm not sure I love either of those though.

Reply:
I'm also not crazy about the name. I think I had `ATPOSTCTORS` in another version. That's probably the most clear.